### ISSUE TITLE
JAWS does not announce the aria-describedby content on textareas when placeholder text is present

### ISSUE THREAD

**Issue by beltonTPG** [FreedomScientific/VFO-standards-support#554]
### Summary

When a `textarea` element has placeholder text, the content of an `aria-describedby` referenced element is not announced. If the `textarea` has an empty placeholder attribute (`placeholder=""`) the `aria-describedby` reference element is announced as expected.

Example:
1. Go to the CodePen example below
2. Tab to the `textarea` elements

### Expected result

JAWS should announce the content of the element referenced by the `aria-describedby` attribute.

### Actual result

JAWS does not announce the content of the element referenced by the `aria-describedby` attribute.

### Example

[CodePen example](https://cdpn.io/brianelton/debug/poPWvaQ/RBMOJRQdmRLk)

#### JAWS version and build number
Version 2021.2105.53 ILM

 
#### Operating System and version
Windows 10 Pro, version 21H1

#### Browser and version:
Google Chrome Version 91.0.4472.124

**Comment by JAWS-test** [comment]
The problem also occurs with Firefox and Edge. It is a JAWS issue because the information in the Accessibility API is correctly transmitted by the browsers. Thus, it does not occur with NVDA either. 
Important: with textarea, the placeholder is also not output, but JAWS only says that the field would contain text (which is not true).

**Comment by StephenBeckettTPGi** [comment]
This issue has now been merged with #553 - closing.
